azure-pipelines.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       matrix:
         # We're only running one platform per Python version to speed up builds
         Python36Linux:
-          imageName: "ubuntu-latest"
+          imageName: "ubuntu-20.04"
```

Comment:
Out of curiosity, why this change?

Reply:
See #11928 - `ubuntu-latest` changed a little earlier than we expected, and it drops 3.6.